### ISSUE TITLE
Do not delete empty marker category on restart

### ISSUE THREAD

**Issue by opk12** [osmandapp/Osmand#9380]
I use a couple of categories "review later", to catch POIs quickly while moving and to sort them later by category (and to upstream them at home, when I can give them more thought, can read the wiki, etc).

When OsmAnd is closed and restarted, or when the phone is restarted, empty categories appear to have been deleted. This is easily avoided with a dummy marker, but I thought I'd report it anyway.

OsmAnd 3.7

**Comment by vshcherb** [comment]
Markers don't have categories if I'm not mistaken. Are you talking about favorites? Are they gone after restart?

**Comment by pebogufi** [comment]
@opk12 
you are right. Empty fav groups are deleted automatically. I also keep them with a dummy.

**Comment by vshcherb** [comment]
How is it possible to create / select empty favorite group?

**Comment by scaidermern** [comment]
> How is it possible to create / select empty favorite group?

When editing an existing POI you can select the group or add new groups. However being able to add empty groups is not really intended, I guess.
